overlord/hookstate/context.go

```diff
@@ -165,7 +206,7 @@ func (c *Context) Get(key string, value interface{}) error {
 
```

Comment:
Sorry, not "hook-context".. that's awkward given the idea here. Maybe "ephemeral-context".